tests/tools/test_sliding_window.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from typing import Any, Literal
+
+import pytest
+from anndata import AnnData
+from squidpy.tl import sliding_window
+from squidpy.tl._sliding_window import _sliding_window_stats
+
+
+class TestSlidingWindow:
+    @pytest.mark.parametrize("library_key", ["point"])
+    @pytest.mark.parametrize("window_size", [10])
+    @pytest.mark.parametrize("overlap", [0, 2])
+    def test_design_matrix_several_slices(
+        self,
+        adata_mibitof: AnnData,
+        library_key: str,
+        window_size: int,
+        overlap: int,
+    ):
+        df = sliding_window(
+            adata_mibitof,
```

Comment:
Would be nicer to evaluate on toy anndata f.e. with a 4x4 point grid, then you can explicitly evaluate on that. That actually also makes me think of the edge case when a cell is right on the border - does your function handle that?